slackbot_using_azure_openai/README.md

```diff
@@ -0,0 +1,38 @@
+# Chatbot service for Slack using Azure OpenAI
+
+## Use Case
+The service responds to questions asked in Slack chanels using the slash command (`/<COMMAND> question`), using the Azure OpenAI chat model by maintainging a history of 25 messages.
+
+## Prerequisites
+* Azure OpenAI service
+* Slack account and app
+
+## Setting up Azure OpenAI service
+1. Follow the instructions given in [Azure Documentation](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource?pivots=web-portal) to create an Azure OpenAI resource and deploy a text completion model.
+2. In the Azure Portal, go to **Keys and Endpoint** in the **Resource Management** section of the resource and obtain the service URL (**Endpoint**) and API key (either **KEY1** or **KEY2**).
+3. Go to **Deployments** in the **Resource Management** section and obtain the name of the text completion model which was deployed in step 1.
+
+## Setting up Slack account and app
+1. Sign up and log into [Slack](https://slack.com/get-started#/createnew).
+2. Create a new [Slack app](https://api.slack.com/apps).
+3. Configure a new slash command (e.g. `/test`) for the app via **Features &rarr; Slash Commands**
```

Comment:
```suggestion
3. Configure a new slash command (e.g., `/test`) for the app via **Features &rarr; Slash Commands**
```

Reply:
Addressed in [6737695](https://github.com/ballerina-guides/ai-samples/pull/64/commits/6737695630388bbd0350796f53bcd230e7b02993)